### DISCUSSION TITLE
SQL Views [Oct 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4167] [Discord Threads]
> 🙋‍♂️ **Flame**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### SQL Views are displayed by default when you connect to external DB. MySQL Screenshot displayed here (notice change in icon when compared to table list)  
![image0](https://cdn.discordapp.com/attachments/1028573423607230525/1028645518081011733/Screenshot_2022-10-09_at_5.59.09_PM.png)  
#  
> 🙋‍♂️ **Flame**  
#### I will need to refresh the database connection by creating a new project?  
#  
> 👨‍🦲 **dstala**  
#### SQL views should appear by default. No need for explicit refresh  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1028573423607230525)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
